src/app/app.routing.ts

```diff
@@ -184,6 +185,10 @@ export const appRoutes: Routes = [
 	{
 		path: 'module-inspector',
 		component: ModuleInspectorComponent
+	},
+	{
+		path: 'gpa',
+		component: GpaComponent
```

Comment:
You should have a custom title for 'MyMICDS - GPA' so that the latter part is capitalized instead of the auto-generated